test/scripts/preconditions/test_preconditions.py

```diff
@@ -1,153 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
We shouldn't delete the tests.

Reply:
right... I should understand it first. Sorry.